onnxruntime/python/tools/kernel_explorer/README.md

```diff
@@ -1,12 +1,37 @@
-# Kernel Explorer 
+# Kernel Explorer
+
 Kernel Explorer hooks up GPU kernel code with a Python frontend to help develop, test, profile, and auto-tune GPU kernels.
 
-## Example Usage
-```
-mkdir build
-cd build
-cmake -D onnxruntime_ROCM_HOME=/opt/rocm ..
-cmake --build .
-pytest ../kernels --verbose
-python ../kernels/vector_add_test.py
+Kernel explorer mainly target for attention based models and the purpose is specific to it at the moment.
+
+## Build
+
+```bash
```

Comment:
Put this in a shell script, and just refer to the script here?

Reply:
We better not do this. The script is not generic enough, this is only an adviced configuration.

Reply:
You don't do this everytime, you only do it if you just clone the repo. The cumbersomeness is well known, and the problem is coming with CMake that we don't have pre-configured setup beside scripts. And for scripts, if we are not setting it up for CI, we shall never check it in, at least for me, I never see this kind of **scripts** show up in a well-managed repo. In bazel world, there is `.bazelrc` serve the purpose, but in cmake world, we just live with it :(...